src/main/java/domain/Width.java

```diff
@@ -0,0 +1,20 @@
+package domain;
+
+public class Width {
+    private static final String ERROR_MIN_WIDTH = "[ERROR] 너비는 1 이상이어야 합니다";
+    private static final int ZERO = 0;
+    private final int width;
+
+    public Width(final int width) {
+        validateWidthSize(width);
+        this.width = width;
+    }
+    private void validateWidthSize(final int value) {
+        if (value <= ZERO) {
```

Comment:
Width 객체가 가질 수 있는 값에 대한 검증이니 상수와 부등호를 수정하면 의미를 더 나타낼 수 있지않을까 생각해봤습니다 😄 

```suggestion
    private static final int VALID_MIN_VALUE = 1;

        if (value < VALID_MIN_VALUE) {
```